test/unittests/parser_test.cpp

```diff
@@ -119,31 +139,35 @@ TEST(parser, custom_section_nonempty)
 
 TEST(parser, functype_wrong_prefix)
 {
-    const auto section_contents = uint8_t{0x01} + "610000"_bytes;
+    const auto section_contents =
+        "01"
+        "610000"_bytes;
     const auto bin = bytes{wasm_prefix} + make_section(1, section_contents);
     EXPECT_THROW(parse(bin), parser_error);
 }
 
 TEST(parser, type_section_larger_than_expected)
 {
-    const auto section_contents = uint8_t{0x01} + functype_void_to_void;
-    const auto bin = bytes{wasm_prefix} + uint8_t{0x01} + uint8_t(section_contents.size() - 1) +
-                     section_contents;
+    const auto section_contents = "01"_bytes + functype_void_to_void;
```

Comment:
There are still a bunch of cases where the `"01"_bytes` is the vector size and we could just use `make_vec`. Is that our preference? Lets sort it in this PR.

Reply:
Yeah I think it would be good.

Reply:
I don't think refactoring existing test brings much added value. It's only worth if you need to modify it anyway or it fails and you don't know why. Otherwise, you will not want to read a test again.

Reply:
Ok I'm fine with not doing it in this PR.

Reply:
(there's also a school of thought that teaches to "read the tests when you need to understand what the code does")